addons/web/static/src/views/list/list_renderer.js

```diff
@@ -833,15 +833,7 @@ export class ListRenderer extends Component {
         const fieldName = column.name;
         const list = this.props.list;
```

Comment:
maybe we should manage the classes o_column_sortable,... also. No need to put that class if the column is not sortable. So we should interrogate the datapoints and improve isSortable in list_renderer